SU2_CFD/src/numerics/flow/convection/centered.cpp

```diff
@@ -76,7 +76,7 @@ CNumerics::ResidualType<> CCentBase_Flow::ComputeResidual(const CConfig* config)
   if (preacc) {
     AD::SetPreaccIn(Normal, nDim);
     AD::SetPreaccIn(V_i, nDim+5); AD::SetPreaccIn(V_j, nDim+5);
-    AD::SetPreaccIn(Lambda_i);    AD::SetPreaccIn(Lambda_j);
+    AD::SetPreaccIn(Lambda_i, Lambda_j);
```

Comment:
ad_structure done as per #583 
I also tried to add some syntax sugar to be able to specify preaccumulation inputs and outputs like this.